src/styles/OurStory.css

```diff
@@ -0,0 +1,49 @@
+.OurStory {
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.endingBlurbContainer {
+  position: relative;
+  background-color: #E8D3F7;
+  width: 73%;
+  font-family: Raleway;
+  font-size: 22pt;
+  text-align: center;
+  font-weight: normal;
+  padding: 20px;
+  border-radius: 5px;
+  display: inline-block;
+  margin: auto;
+  position: relative;
+  vertical-align: middle;
+  height: 80px;
+}
+
+.fillRightBlurbImages { /* For fortissimo logo */
+  display: inline-block;
+  width: 110px;
+  height: 110px;
+  padding: 0px;
+  border-radius: 80px;
+  box-shadow: 5px 5px 5px grey;
+  margin-right: 2.5vw;
+  background-color: white;
+  vertical-align: middle;
+  text-align: center;
+  position: relative;
+}
+
+.fillLeftBlurbImages { /* For sf.jpg */
+  display: inline-block;
+  width: 110px;
+  height: 110px;
+  padding: 0px;
+  border-radius: 80px;
+  box-shadow: 5px 5px 5px grey;
+  margin-left: 2.5vw;
+  background-color: white;
+  vertical-align: middle;
+  text-align: center;
+  position: relative;
+}
```

Comment:
OurStory.css: Code is clean, good incorporation of class properties for OurStory.css, and well-organized for future additions/amendments.